### PR TITLE
ACMS-3656: Entity canonical issue for headless.

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**

Fixes #ACMS-3656

**Proposed changes**
Fixes headless starter kit installation

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_common/composer.json [hunk 1]
```diff
@@ -93,6 +93,9 @@
         "patches": {
             "drupal/core": {
                 "3301692 - Passing null to parameter to mb_strtolower() is deprecated with PHP 8.1": "https://git.drupalcode.org/project/drupal/-/merge_requests/2598.patch"
+            },
+            "drupal/workbench_email": {
+                "3427515 - Route entity.workbench_email_template.canonical does not exist": "https://git.drupalcode.org/project/workbench_email/-/merge_requests/20.patch"
             }
```
Comment (rajeshreeputra) [inline modules/acquia_cms_common/composer.json:99]: @deepakmishra2 please move this patch in Acquia CMS Common.

File: modules/acquia_cms_common/composer.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
         "drupal/smart_trim": "^2.0",
         "drupal/social_media_links": "^2.9",
         "drupal/username_enumeration_prevention": "^1.3",
-        "drupal/workbench_email": "^3.0",
+        "drupal/workbench_email": "~3.0.3",
```
Comment (vishalkhode1) [inline modules/acquia_cms_common/composer.json:33]: @deepakmishra2 Please pin the version.

File: modules/acquia_cms_common/composer.json (hunk 1 above)
Comment (rajeshreeputra) [reply modules/acquia_cms_common/composer.json:99]: Resolved

File: modules/acquia_cms_common/composer.json (hunk 2 above)
Comment (deepakmishra2) [reply modules/acquia_cms_common/composer.json]: done

**Review by vishalkhode1 — APPROVED** [review]

**Review by rajeshreeputra — APPROVED** [review]

**Comment by deepakmishra2** [comment]
Closing this PR , as workbench_email has released latest version with fixes from MR https://www.drupal.org/project/workbench_email/issues/3427515

**Pull request closed without merging**